### ISSUE TITLE
Visualising molecules and data

### ISSUE THREAD

**Issue by drc007** [OpenSourceMalaria/OSM_To_Do_List#313]
Many thanks to all who helped get the structural information and associated data into the Google spreadsheet. The next stage is to think about how we can leverage that resource to better view project data.
@lpatiny has developed a brilliant web-based interface for visualisation.

http://www.cheminfo.org/flavor/malaria/ <http://www.cheminfo.org/flavor/malaria/

We now need to think about what views of the data people would like to have, perhaps just a table, a 2D scatterplot, a 3D rotatable plot etc.

**Comment by Quinten-U-SAR** [comment]
Hi,
I'm Quentin, head of chemistry with Quinten a company specialized in data science. We have developed unique algorithm to extract hidden signals in multi-parametric projects in chemistry. More over we also have developed an app call U-SAR, in SaaS mode, to visualize medicinal chemistry database, to monitor and to communicate around a project. If you are interested to know more about our services feel free to contact me, I would be more than happy to help you for free !

**Comment by mattodd** [comment]
Great - please come along to the meeting on Monday #317 or check the recording after the meeting is done.

**Comment by Quinten-U-SAR** [comment]
Thx, please let me know were is the recording link, I arrived to late for the meeting... I'm leaving on Wednesday on vacations until the end of august, so If there is still an interest I can do a demo during a meeting in September and then propose several licenses for chemist in charge of the projects.

**Comment by Quinten-U-SAR** [comment]
Hi ! 
still interested in a data vizualisation app in saas mode ?
Let me know who might be interested and then I can contact him directly !
best regards
Quentin

**Comment by lpatiny** [comment]
We have updated the website to display the data:
http://www.cheminfo.org/flavor/malaria/Display_data.html
What's new:
* by default only the predicted values are displayed. The biological data are too sparse so that it can be useful for filtering
* color in the parallel coordinates now depends of the predicted value of logP
* mouse click on a line display all the information available for this compound

What should be done next week is to display for each information the detailed information on how it was calculated based on the other spreadsheet. Should be done this week

**Comment by mattodd** [comment]
Sure - @lpatiny the only default bio value I'd say we need to show at this stage is mean potency - that's what most people are interested in up-front, and then people would go searching for the other bio data if needed. Can we include that by default, if we made sure that an average is calculated in the relevant cell in the spreadsheet?

**Comment by drc007** [comment]
We need to populate the spreadsheet with the series 1 bioactivity, if it was generated in a different lab or protocol then it should be in a new column. Then someone @mattodd needs decide what biological data is comparable and if we can reasonably have a single "mean potency" field. I'm afraid I don't know enough about the specific assays to make that judgement.

**Comment by mattodd** [comment]
Agreed, those data need to be completed next in readiness for paper submission, though it's an option we submit first and complete as the paper is wending its way. Yes, several different labs, all of which I think already have columns in the sheet.

**Comment by lpatiny** [comment]
What is the name of the column ? PfaI EC50 (Inh) ? And I should put what as title of the column ? Mean potency ?

**Comment by mattodd** [comment]
Column Q - the one with the word "Mean" in the title. Yes, "Mean Potency".
We can finesse this over time. It may be that we are averaging potencies
against different strains, but let's do this for now.

One nagging concern. In the spreadsheet of assays there are URLs to the ELN
pages that contain the raw data but at this stage the numbers in the
primary data sheet are not linked to a specific source, so people have to
hunt a little. I think that's OK now, but obviously that becomes annoying
if there are, later, 10 URLs for a particular assay and one needs to go
hunting.


On 20 October 2015 at 00:49, lpatiny <notifications@github.com> wrote:

> What is the name of the column ? PfaI EC50 (Inh) ? And I should put what
> as title of the column ? Mean potency ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OpenSourceMalaria/OSM_To_Do_List/issues/313#issuecomment-149218630>
> .
>



-- 
MATTHEW TODD | Associate Professor
School of Chemistry | Faculty of Science

THE UNIVERSITY OF SYDNEY
Rm 519, F11 | The University of Sydney | NSW | 2006
T +61 2 9351 2180  | F +61 2 9351 3329  | M +61 415 274104
E matthew.todd@sydney.edu.au | W
http://sydney.edu.au/science/people/matthew.todd.php
W http://opensourcemalaria.org/ | W http://opensourcetb.org/ | W
http://opensourcepharma.net/

CRICOS 00026A
This email plus any attachments to it are confidential. Any unauthorised
use is strictly prohibited. If you receive this email in error, please
delete it and any attachments.

**Comment by lpatiny** [comment]
@mattodd  @drc007 
The webpage has been updated in order to be able to export any selection of data from the spreadsheet as a SDF or tab-delimited.
On the top of each column you may put a query like >1, 1 or 1..3 (between one and 3). You may then click on the top of the first column to select all the found products and then click on
![image](https://cloud.githubusercontent.com/assets/1484241/15349163/af337c74-1cc8-11e6-8061-a7704d02548c.png)



![image](https://cloud.githubusercontent.com/assets/1484241/15349147/89615ebc-1cc8-11e6-91c1-bb9ff631531e.png)

**Comment by mattodd** [comment]
Oh that's fantastic, Luc, thank you so much. Perfectly in time for the
meeting (#386). Chris, would you mind checking that this works as an import
to e.g. DataWarrior? Shall we use this as the SDF generator, or do you see
a little additional value in posting a pre-done SDF somewhere (a slight
lowering of the barrier).

On 18 May 2016 at 16:18, lpatiny <notifications@github.com> wrote:

> @mattodd <https://github.com/mattodd> @drc007 <https://github.com/drc007>
> The webpage has been updated in order to be able to export any selection
> of data from the spreadsheet as a SDF or tab-delimited.
> On the top of each column you may put a query like >1, 1 or 1..3 (between
> one and 3). You may then click on the top of the first column to select all
> the found products and then click on
> [image: image]
> <https://cloud.githubusercontent.com/assets/1484241/15349163/af337c74-1cc8-11e6-8061-a7704d02548c.png>
>
> [image: image]
> <https://cloud.githubusercontent.com/assets/1484241/15349147/89615ebc-1cc8-11e6-91c1-bb9ff631531e.png>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/OpenSourceMalaria/OSM_To_Do_List/issues/313#issuecomment-219935201>
>



-- 
MATTHEW TODD | Associate Professor
School of Chemistry | Faculty of Science

THE UNIVERSITY OF SYDNEY
Rm 519, F11 | The University of Sydney | NSW | 2006
T +61 2 9351 2180  | F +61 2 9351 3329  | M +61 415 274104
E matthew.todd@sydney.edu.au | W
http://sydney.edu.au/science/people/matthew.todd.php
W http://opensourcemalaria.org/ | W http://opensourcetb.org/ | W
http://opensourcepharma.net/

CRICOS 00026A
This email plus any attachments to it are confidential. Any unauthorised
use is strictly prohibited. If you receive this email in error, please
delete it and any attachments.

**Comment by drc007** [comment]
@lpatiny @mattodd This looks great. However if I use this link 

http://www.cheminfo.org/flavor/malaria/Display_data.html

I don't get the option to download SDF selected? If I click on the Display Data menu I do get it. I can then select the series and download an SDF file which opens fine and contains all the data.

It might be worth having a copy of the SDF available for those users that don't use Chrome.

**Comment by lpatiny** [comment]
The correct link is http://www.cheminfo.org/flavor/malaria/Display_data/index.html  it is the one that your reach from : http://malaria.cheminfo.org
I updated the link from the google spreadsheet that was pointing to the old version.
The page was also updated to be compatible with https.

**Comment by drc007** [comment]
Brilliant, thanks.

**Comment by mattodd** [comment]
Shortened the link to cheminfo to http://tinyurl.com/OSM-Cheminfo for the purposes of the meeting. Also posted the SDF to [here](http://malaria.ourexperiment.org/data/files/17162/malaria-2016-05-18.sdf) for download.

**Comment by lpatiny** [comment]
You know you have 

http://malaria.chemifo.org

That is even shorter

You may also edit the homepage there is you want.

**Comment by mattodd** [comment]
Yes, indeed - it's just that I know some people don't realise they need to
click on "Display Data" to see anything ;) I think people expect the page
they land on to have chemical structures...

On 18 May 2016 at 22:01, lpatiny <notifications@github.com> wrote:

> You know you have
>
> http://malaria.chemifo.org
>
> That is even shorter
>
> You may also edit the homepage there is you want.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/OpenSourceMalaria/OSM_To_Do_List/issues/313#issuecomment-220005506>
>



-- 
MATTHEW TODD | Associate Professor
School of Chemistry | Faculty of Science

THE UNIVERSITY OF SYDNEY
Rm 519, F11 | The University of Sydney | NSW | 2006
T +61 2 9351 2180  | F +61 2 9351 3329  | M +61 415 274104
E matthew.todd@sydney.edu.au | W
http://sydney.edu.au/science/people/matthew.todd.php
W http://opensourcemalaria.org/ | W http://opensourcetb.org/ | W
http://opensourcepharma.net/

CRICOS 00026A
This email plus any attachments to it are confidential. Any unauthorised
use is strictly prohibited. If you receive this email in error, please
delete it and any attachments.

**Comment by lpatiny** [comment]
OK I redirected the page to the data so you may advertise

http://malaria.cheminfo.org <http://malaria.cheminfo.org/>
